centreon-engine/src/main.cc

```diff
@@ -173,35 +174,65 @@ int main(int argc, char* argv[]) {
+             "Copyright 2009-2010 Nagios Core Development Team and Community "
              "Contributors\n"
-          << "Copyright 2011-2021 Centreon\n"
-          << "\n"
-          << "This program is free software: you can redistribute it and/or\n"
-          << "modify it under the terms of the GNU General Public License "
+             "Copyright 2011-2021 Centreon\n"
+             "\n"
+             "This program is free software: you can redistribute it and/or\n"
+             "modify it under the terms of the GNU General Public License "
              "version 2\n"
-          << "as published by the Free Software Foundation.\n"
-          << "\n"
-          << "Centreon Engine is distributed in the hope that it will be "
+             "as published by the Free Software Foundation.\n"
+             "\n"
+             "Centreon Engine is distributed in the hope that it will be "
              "useful,\n"
-          << "but WITHOUT ANY WARRANTY; without even the implied warranty of\n"
-          << "MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the "
+             "but WITHOUT ANY WARRANTY; without even the implied warranty "
+             "of\n"
+             "MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the "
              "GNU\n"
-          << "General Public License for more details.\n"
-          << "\n"
-          << "You should have received a copy of the GNU General Public "
+             "General Public License for more details.\n"
+             "\n"
+             "You should have received a copy of the GNU General Public "
              "License\n"
-          << "along with this program. If not, see\n"
-          << "<http://www.gnu.org/licenses/>.";
+             "along with this program. If not, see\n"
+             "<http://www.gnu.org/licenses/>.";
+
+      log_v2::config()->info(
+          "Centreon Engine {} \n"
+          "\n"
+          "Copyright 1999-2009 Ethan Galstad\n"
+          "Copyright 2009-2010 Nagios Core Development Team and Community "
+          "Contributors\n"
+          "Copyright 2011-2021 Centreon\n"
+          "\n"
+          "This program is free software: you can redistribute it and/or\n"
+          "modify it under the terms of the GNU General Public License "
+          "version 2\n"
+          "as published by the Free Software Foundation.\n"
+          "\n"
+          "Centreon Engine is distributed in the hope that it will be "
+          "useful,\n"
+          "but WITHOUT ANY WARRANTY; without even the implied warranty "
+          "of\n"
+          "MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the "
+          "GNU\n"
+          "General Public License for more details.\n"
+          "\n"
+          "You should have received a copy of the GNU General Public "
+          "License\n"
+          "along with this program. If not, see\n"
+          "<http://www.gnu.org/licenses/>.",
+          CENTREON_ENGINE_VERSION_STRING);
+
```

Comment:
std::cout looks better than log_v2 here. The goal of these messages is to display them in the terminal